mysql-test/suite/galera/t/galera_garbd_backup.test

```diff
@@ -0,0 +1,65 @@
+#
+# Test a single node with an arbitrator.  Startup a 2-node cluster, shut down node 2, and use it's ports
```

Comment:
Is this comment relevant? I can't see shutting down node 2 during the test.
Node's 3 port seems to be used instead.

Reply:
Removed.